docs/java-rest/high-level/apis/index.asciidoc

```diff
@@ -1,10 +1,21 @@
 include::createindex.asciidoc[]
+
 include::deleteindex.asciidoc[]
+
+include::close_index.asciidoc[]
+
 include::_index.asciidoc[]
+
 include::get.asciidoc[]
+
 include::delete.asciidoc[]
+
 include::update.asciidoc[]
+
 include::bulk.asciidoc[]
+
 include::search.asciidoc[]
+
 include::scroll.asciidoc[]
+
```

Comment:
do we need these extra empty lines?

Reply:
not really... A good example is 
https://github.com/elastic/elasticsearch/blob/master/docs/java-api/admin/indices/index.asciidoc ( where both approaches are present ;) ) 
Basically if there is no extra new line, the two terms are displayed on the same line.